x-pack/plugins/ui_actions_enhanced/public/components/action_wizard/action_wizard.tsx

```diff
@@ -99,13 +144,85 @@ export const ActionWizard: React.FC<ActionWizardProps> = ({
   );
 };
 
-interface SelectedActionFactoryProps {
+interface TriggerPickerProps {
+  triggers: TriggerId[];
+  selectedTriggers?: TriggerId[];
+  getTriggerInfo: (triggerId: TriggerId) => Trigger;
+  onSelectedTriggersChange: (triggers?: TriggerId[]) => void;
+  triggerPickerDocsLink?: string;
+}
+
+const TriggerPicker: React.FC<TriggerPickerProps> = ({
```

Comment:
I would like do that as soon as we need `TriggerPicker` somewhere else? 
frankly, It is very convenient to for me to keep those piece in 1 file since most of the time new changes are affecting all of those